OrbitCore/Capture.cpp

```diff
@@ -31,10 +31,10 @@ std::shared_ptr<PresetFile> Capture::GSessionPresets = nullptr;
 
 void Capture::Init() { GTargetProcess = std::make_shared<Process>(); }
```

Comment:
nit: you can remove brackets around (b/42) and if you add http:// before you will be able to open it from console while looking at git log (as it is now you will need to copy and paste it)

Reply:
Sure. Done!

Reply:
ehm, test does not describe how to test this change. I will not be able to reproduce it.